scipy/stats/_stats_py.py

```diff
@@ -9440,3 +9442,88 @@ def rankdata(a, method='average', *, axis=None, nan_policy='propagate'):
         result[nan_indexes] = np.nan
 
     return result
+
+
+# TODO: axis parameter
+# TODO: add _axis_nan_policy_factory decorator
+def expectile(a, alpha=0.5, axis=0, dtype=None, weights=None):
```

Comment:
I can see `alpha` allowing positional arguments, but probably everything after should be keyword-only.